### PR TITLE
update portfolio page

### PR DESCRIPTION
**Pull request by BaloneyBoy97**
tech lead's data updated #111
Description: Made changes in the tech leads information on the project's portfolio page.

what changes were made?
Tech leads information is updated on the project's portfolio page.

Why were changes made?
The changes were made in the tech lead position.

How was it changed?
Made some changes in the about. md file in the project portfolio and added.

Testing Done:
cross-checked the content on the portfolio page by running in localhost.
Screenshot 2024-08-30 at 5 17 50 PM

### CODE REVIEW

File: docs/project_wheres_religion_desktop/about.md [hunk 1]
```diff
@@ -13,11 +13,12 @@ Where's Religion? is conceptualized and designed for diverse users with interest
...
+- **Senior Tech Lead:** [Yash Bhatia](https://yashb196.github.io/yashb196/) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/yashb196) [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/yashbhatia238/)
+- **Current Tech Lead:** Zanxiang Wang [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/BaloneyBoy97) [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/zanxiang-wang-352b112a0/)
 - **Developers:**
-  - Stuart Ray (capstone) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/Stuartwastaken)
-  - Izak Robles (capstone) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/izakrobles)
-  - Josh Hogan (capstone) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/Josh-Hogan-87)
+  - Stuart Ray (Alumni) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/Stuartwastaken)
```
Comment (kungfuchicken) [inline docs/project_wheres_religion_desktop/about.md:19]: the "alumni" tag should be lowercase.

File: docs/project_wheres_religion_desktop/about.md [hunk 2]
```diff
@@ -13,11 +13,12 @@ Where's Religion? is conceptualized and designed for diverse users with interest
 
 - **Source Code:** [https://github.com/oss-slu/lrda_website](https://github.com/oss-slu/lrda_website) [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/oss-slu/lrda_website)
 - **Client** Dr. Adam Park
-- **Current Tech Lead:** [Yash Bhatia](https://yashb196.github.io/yashb196/) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/yashb196) [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/yashbhatia238/)
+- **Senior Tech Lead:** [Yash Bhatia](https://yashb196.github.io/yashb196/) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/yashb196) [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/yashbhatia238/)
```
Comment (kungfuchicken) [inline docs/project_wheres_religion_desktop/about.md:16]: The mobile page has @yashb196 listed as "Senior Developer". I don't care which title we use, but we need to be consistent between these projects.

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
almost there!

File: docs/project_wheres_religion_desktop/about.md (hunk 1 above)
Comment (BaloneyBoy97) [reply docs/project_wheres_religion_desktop/about.md]: issue fixed

File: docs/project_wheres_religion_desktop/about.md (hunk 2 above)
Comment (BaloneyBoy97) [reply docs/project_wheres_religion_desktop/about.md]: issue fixed

**Review by kungfuchicken — APPROVED** [review]
lgtm;

**Pull request merged**